tests/foreman/cli/test_organization.py

```diff
@@ -115,142 +113,62 @@ def test_verify_bugzilla_1078866(self):
         self.assertEqual(len(set(lines)), len(lines))
 
     @tier1
-    def test_positive_create_with_name(self):
-        """Create organization with valid name only
-
-        :id: 35840da7-668e-4f78-990a-738aa688d586
-
-        :expectedresults: organization is created and has appropriate name
-
-        :CaseImportance: Critical
-        """
-        for name in valid_org_names_list():
-            with self.subTest(name):
-                org = make_org({'name': name})
-                self.assertEqual(org['name'], name)
-
-    @tier1
-    def test_positive_create_with_matching_name_label(self):
-        """Create organization with valid matching name and label only
-
-        :id: aea551de-145b-4894-b4fb-65878ff1f101
-
-        :expectedresults: organization is created, label matches name
-
-        :CaseImportance: Critical
-        """
-        for test_data in valid_labels_list():
-            with self.subTest(test_data):
-                org = make_org({
-                    'label': test_data,
-                    'name': test_data,
-                })
-                self.assertEqual(org['name'], org['label'])
-
-    @tier1
-    def test_positive_create_with_unmatched_name_label(self):
-        """Create organization with valid unmatching name and label only
-
-        :id: a4730b09-1bd7-4b00-a7ee-76080a916ea8
-
-        :expectedresults: organization is created, label does not match name
-
-        :CaseImportance: Critical
-        """
-        for name in valid_org_names_list():
-            with self.subTest(name):
-                label = gen_string('alpha')
-                org = make_org({
-                    'label': label,
-                    'name': name,
-                })
-                self.assertNotEqual(org['name'], org['label'])
-                self.assertEqual(org['name'], name)
-                self.assertEqual(org['label'], label)
-
-    @tier1
-    def test_positive_create_with_name_description(self):
-        """Create organization with valid name and description only
-
-        :id: b28c95ba-918e-47fe-8681-61e05b8fe2ea
-
-        :expectedresults: organization is created
-
-        :CaseImportance: Critical
-        """
-        for name, desc in zip(valid_org_names_list(), valid_data_list()):
-            with self.subTest(name + desc):
-                org = make_org({
-                    'description': desc,
-                    'name': name,
-                })
-                self.assertEqual(org['name'], name)
-                self.assertEqual(org['description'], desc)
-
-    @tier1
-    @upgrade
-    def test_positive_create_with_name_label_description(self):
+    def test_positive_CRD(self):
```

Comment:
typo `CRD` => `CRUD` ?

Reply:
well, I do the update tests separately (including also those would make the test too long), therefore no U :)